packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -168,7 +171,8 @@ class IconButton extends StatelessWidget {
     return new InkResponse(
       onTap: onPressed,
       child: result,
-      radius: math.max(size, Material.defaultSplashRadius),
+      radius: math.max(48.0, iconSize + math.min(padding.horizontal, padding.vertical))
+          * 0.55, // x 0.5 for diameter -> radius and + 10% overflow so it doesn't look too square.
```

Comment:
> Material.defaultSplashRadius is a static number right?

Yes.

> I didn't find anything specific in the material guidelines.

Ok, I think I found my source.  Check out this video:

https://storage.googleapis.com/material-design/publish/material_v_10/assets/0B14F_FSUCc01MmtYUVI0VE85Y00/02_SearchRipple-v2.webm

Here's a screen capture when the "inner" splash on the right reaches its largest radius:

![inkreaction](https://cloud.githubusercontent.com/assets/112007/23196205/0f831758-f86d-11e6-94f0-459beeefe48b.png)

In the video, the radius of that splash (not the largest pink circle but the one inside that one (not the darkest pink ring), is 56.0 pixels.  However, the video doesn't appear to be 1-to-1 because the toolbar is 90.0 pixels.  We know from another part of the spec that the toolbar is actually 56.0 logical pixels, so that puts the radius of the splash at 56/90*56 = 34.8 logical pixels (which I think we rounded to 35.0).